snippets/24-use-switch-case-with-numeric-ranges.js

```diff
@@ -0,0 +1,13 @@
+// Minutes of Javascript #24 - Use switch/case statement with numeric ranges
+
+//You can use a switch statement with a numeric range in the case with:
```

Comment:
```suggestion
// switch (true) may seem absurd but you can match against values as well as expressions. 
// Expressions will be evaluated before matching, so any expression that evals to true will be matched.
// Ref: https://262.ecma-international.org/10.0/#sec-switch-statement

// For example, you can use a switch statement with a numeric range in the case with:
```

Reply:
Ah, this description makes so much more sense than mine, thanks!! 👍🏻